ui/apps/platform/cypress/constants/apiEndpoints.js

```diff
@@ -71,6 +71,12 @@ export const integrationHealth = {
     vulnDefinitions: '/v1/integrationhealth/vulndefinitions?component=*',
 };
 
+export const credentialHealth = {
+    central: '/v1/credentialexpiry?component=CENTRAL',
+    centralDb: '/v1/credentialexpiry?component=CENTRAL_DB',
+    scanner: '/v1/credentialexpiry?component=SCANNER',
+};
+
```

Comment:
Although not a blocker to merge, we have been encapsulating endpoint in the integration test folders.

Although any tests for main banners will need these, no reason against putting them in systemHealth too.

Reply:
Not sure I follow, can you point to an example of what you mean?

Reply:
Like this: https://github.com/stackrox/stackrox/pull/3603/files